README.md

```diff
@@ -30,66 +31,68 @@ END_POINT = "pr.oxylabs.io:7777"
 
 async def fetch():
     async with aiohttp.ClientSession() as session:
-	    async with session.get("http://ip.oxylabs.io", proxy=f"http://{USER}:{PASSWORD}@{END_POINT}") as resp: 
-		print(await resp.text())
+        async with session.get("http://ip.oxylabs.io", 
+            proxy=f"http://{USER}:{PASSWORD}@{END_POINT}") as resp: 
+            print(await resp.text())
 ```
 In order to use your own proxies, adjust `user` and `pass` fields with your Oxylabs account credentials.
 
 ## Testing Proxies
-To check if the proxy is working correctly, try visiting http://ip.oxylabs.io/. It will return your current IP address.
+To see if the proxy is working, try visiting https://ip.oxylabs.io. If everything is working correctly, it will return an IP address of a proxy that you're currently using.
 
 ## Sample Project: Extracting Data From Multiple Pages
 To better understand how residential proxies can be utilized for asynchronous data extracting operations, we wrote a sample project to scrape product listing data and save the output to a `CSV` file. The proxy rotation allows us to send multiple requests at once risk-free – meaning that we don't need to worry about CAPTCHA or getting blocked. This makes the web scraping process extremely fast and efficient – now you can extract data from thousands of products in a matter of seconds!
 ```python
-import aiohttp
 import asyncio
+import time
+import sys
+
+import aiohttp
 from bs4 import BeautifulSoup
 import pandas as pd
-import sys
-import time
 
 USER = "user"
 PASSWORD = "pass"
 END_POINT = "pr.oxylabs.io:7777"
 
 # Generate a list of URLs to scrape
-url_list = [f"https://books.toscrape.com/catalogue/category/books_1/page-{page_num}.html" for page_num in range(1, 51)]
+url_list = [f"https://books.toscrape.com/catalogue/category/books_1/page-{page_num}.html" \
+for page_num in range(1, 51)]
 
 async def fetch(session, sem, url):
-	async with sem:
-		async with session.get(url, proxy=f"http://{USER}:{PASSWORD}@{END_POINT}") as response: 
-			await parse_url(await response.text())
+    async with sem:
+        async with session.get(url, proxy=f"http://{USER}:{PASSWORD}@{END_POINT}") as response: 
+            await parse_url(await response.text())
 
 async def parse_url(text):
-	soup = BeautifulSoup(text, "lxml")
-	for product_data in soup.select("ol.row > li > article.product_pod"):
-		data = {
-		"Title": product_data.select_one("h3 > a")["title"],
-		"URL": product_data.select_one("h3 > a").get("href")[5:],
-		"Product Price": product_data.select_one("p.price_color").text,
-		"Stars": product_data.select_one("p")["class"][1],
-		}
-		full_data.append(data)
-		print(f"Grabbing book: {data['Title']}")
-	
+    soup = BeautifulSoup(text, "lxml")
+    for product_data in soup.select("ol.row > li > article.product_pod"):
+        data = {
```

Comment:
```python
data = {
   "Title": product_data.select_one("h3 > a")["title"],
   "URL": product_data.select_one("h3 > a").get("href")[5:],
   "Product Price": product_data.select_one("p.price_color").text,
   "Stars": product_data.select_one("p")["class"][1],
}
```
Properly indent inside data structures.

I'd suggest using technical-sounding names for the dict keys, like `title`, `url` and `product_price`, but that is not essential.